pkg/minishift/util/setproxytoshell.go

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
@budhrg In go for file name and variable we don't use "_" but use Mixed-case IIRC.